doc/manual/src/contributing/json-guideline.md

```diff
@@ -0,0 +1,132 @@
+# JSON guildeline
+
+Nix consumes and produces JSON in a variety of contexts.
+These guidelines ensure consistent practices for all our JSON schema, so it is easy to communicate with Nix, and intuition gained from working with schemas transfer to working with another.
```

Comment:
```suggestion
These guidelines ensure consistent practices for all our JSON interfaces, for ease of use, and so that experience in one part carries over to another.
```